Gemfile

```diff
@@ -1,5 +1,5 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
-puppet_branch = ENV['puppet_branch'] || "latest"
+puppet_branch = ENV['PUPPET_VERSION'] || "latest"
```

Comment:
when this goes in we'll need to update our spec tests in jenkins to use this. Easy change though and the name is better.